atom/app/resources/mac/loginhelper-Info.plist

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleIdentifier</key>
+	<string>${ATOM_BUNDLE_ID}</string>
```

Comment:
This needs to be another id, separate of the main-app, that is also registered with apple.

Reply:
[`ATOM_BUNDLE_ID` is `com.github.electron.loginhelper`.](https://github.com/dittos/electron/blob/4119da607fbebc14bed98cc398e7272a37b3b57c/electron.gyp#L720) It should be changed anyway, but there is no good way to automatically customize it. (as prebuilt binaries are distributed unbranded - may go into third party tools like electron-builder)